test/Conversion/ExportVerilog/max-terms.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: circt-opt -lowering-options=maximumNumberOfTermsPerExpression=4 --export-verilog %s | FileCheck %s
+
+// CHECK-LABEL: module large_use_in_procedural
+hw.module @large_use_in_procedural(%a: i1) {
+  // CHECK: wire _GEN;
+  // CHECK: assign _GEN = a + a + a + a + a;
```

Comment:
shouldn't this be two expressions since it has 5 terms?

Reply:
In the IR it is split, but then export goes ahead and inlines the split expressions into one line.